x/consensus/migrations/v2/store.go

```diff
@@ -0,0 +1,42 @@
+package v2
```

Comment:
why do we have a v2? If it's only to run migrations from the old param store, it seems hacky to me.

Conceptually, populating the initial state should be done in InitGenesis

Reply:
this was added by @JeancarloBarrios. 

How would init genesis work if the store is being migrated. On restart of the node initgenesis Is called so like that or?

Reply:
Yes, when a new module is introduced, the upgrade handlers calls its InitGenesis instead of the migrations.

Reply:
then the approach will need to change with base app and how base app calls this.

Reply:
how the module is initialised will come in the pr that ties this into base app. I believe base app needs to get the old values in init and set them in the new module. This is basically the initgenesis of this module.